src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -122,6 +122,20 @@ private DslPart createResponseForGetTask() throws JsonProcessingException {
                         .stringType("case_name", "Bob Smith")
                         .booleanType("warnings", false)
                         .stringType("work_type", "hearing_work")
+                    // Note: Array verification cannot be done since unorderedArray is introduced in v4 Specification
+                    // See: RWA-900
```

Comment:
are we not better off just removing this code and have the comment in the commit history?

Reply:
yes you're right I'll just remove it